src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleViewModel.java

```diff
@@ -0,0 +1,120 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.gui.errorconsole;
+
+import java.util.Random;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.logging.GuiAppender;
+
+import javafx.beans.property.ReadOnlyStringProperty;
+import javafx.beans.property.ReadOnlyStringWrapper;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+public class ErrorConsoleViewModel {
+
+    //maybe make it editable as a preference?
+    private static final long REFRESH_RATE = 1000;
+
+    private Thread refreshThread = new Thread(() -> {
+        boolean injectedCloseButton = false;
+        try {
+            while (true) {
+                Thread.sleep(REFRESH_RATE);
+
+                // IMO this is a really, really bad way to do this, but a saw no other way since the stage is
```

Comment:
how can i bind the observable list to this outputStream. The problem is that all output gets written to a stream which in my opinion is incompatibe with a observable List.